src/PBot.Tests.Integration/Helper.cs

```diff
@@ -5,27 +5,35 @@
 //copy pasted from Octokit.net
 public static class Helper
 {
-    static readonly Lazy<Credentials> _credentialsThunk = new Lazy<Credentials>(() =>
```

Comment:
Immutability of the containing type is also only one reason to use `readonly`. If a field should never be reassigned, even if it is mutated internally by the type, I also believe that should be expressed.